garden-backend-service/src/api/routes/_utils.py

```diff
@@ -110,3 +113,26 @@ async def poll_globus_search_task(
         raise exceptions.HTTPException(
             status.HTTP_500_INTERNAL_SERVER_ERROR, detail=task_result.text
         )
+
+
+def _garden_sqlalchemy_to_pydantic(garden: Garden):
```

Comment:
see above -- I don't think we need this if `model_validate` works instead. IIRC `model_validate` is what fastAPI calls under the hood when your route returns a plain python object to parse it into the response body